egret/models/dcopf_losses.py

```diff
@@ -158,7 +170,7 @@ def create_btheta_losses_dcopf_model(model_data, relaxation_type=RelaxationType.
     libbranch.declare_ineq_p_branch_thermal_lbub(model=model,
                                                  index_set=branch_attrs['names'],
                                                  branches=branches,
-                                                 p_thermal_limits=p_max,
+                                                 p_thermal_limits=branch_p_max,
```

Comment:
Shouldn't this be `pf_bounds`? I.e., this behavior for the thermal limits is identical to before.

I think this would need to be corrected in `libbranch.declare_ineq_p_branch_thermal_lbub`, which a one point negates the values in the dictionary passed into `p_thermal_limits`. (Which I'm guessing is the issue from PR#128).